wireless_watcher/scripts/watcher_node

```diff
@@ -145,7 +148,7 @@ def main():
                 previous_error = False
                 rospy.loginfo("Retrieved status of interface %s. Now updating at %f Hz." % (dev, hz))
 
-        except subprocess.CalledProcessError:
+        except:
```

Comment:
Apparently I'm subscribed to this and so I got emails about this PR even though I'm not on it. Got curious and took a look and want to share some unrequested feedback.

Typically naked `except:` is not what you want, to the extent that it's usually a linting error.  If you _do_ want to capture everything, use `except Exception:`. It's actually different because `Exception` doesn't include some exceptions like KeyboardInterrupt and SystemExit.  You can see a really nice hierarchy here: https://docs.python.org/3/library/exceptions.html#exception-hierarchy

And if the answer is. "Yuuup I knew that and want to capture those exceptions too" well then  `homer_receeding _into_the_bushes.gif`

Reply:
I just did a cursory readup on exceptions when I was writing this, and saw that not all exceptions are guaranteed to inherit from `Exception`, so I just though, "well, might as well get everything." But I am happy to catch everything below `Exception` instead.

Reply:
(Just FYI, the linter was cool with this)

Reply:
Cool. In Python tradition there's like 5 linters and countless settings for them.  Mine complains but I imagine the one we run in CI doesn't.

No big deal either way, I'm not saying you must change this.  Just wanted to share the segment of knowledge.